packages/firebase_crashlytics/lib/src/firebase_crashlytics.dart

```diff
@@ -41,11 +41,8 @@ class Crashlytics {
           Trace.format(details.stack).trimRight().split('\n');
       final List<Map<String, String>> stackTraceElements =
           _getStackTraceElements(stackTraceLines);
-      final dynamic result =
-          // TODO(amirh): remove this on when the invokeMethod update makes it to stable Flutter.
-          // https://github.com/flutter/flutter/issues/26431
-          // ignore: strong_mode_implicit_dynamic_method
-          await channel.invokeMethod('Crashlytics#onError', <String, dynamic>{
+      final String result =
```

Comment:
@kroikie I saw that it returns string from the platform so I changed it to String. Please confirm it is correct.

Reply:
It actually fails a test. I reverted to have dynamic. I'll let you to update it if need to.